content/src/service/synchronization/ContentCluster.ts

```diff
@@ -118,6 +118,10 @@ export class ContentCluster {
                         // Create new client
                         ContentCluster.LOGGER.info(`Could re-connect to server ${newName} on ${address}`)
                         newClient = getClient(this.fetchHelper, address, newName, previousClient.getEstimatedLocalImmutableTime())
+                    } else if (previousClient.getName() !== newName) {
+                        // Update known name to new one
+                        ContentCluster.LOGGER.warn(`Server's name changed on ${address}. It was ${previousClient.getName()} and know it is ${newName}.`)
+                        newClient = getClient(this.fetchHelper, address, newName, previousClient.getEstimatedLocalImmutableTime())
```

Comment:
should we stop de previousClient? is that already being done?

Reply:
No need to stop them, they don't do nothing by themselves